pkgs/applications/networking/mailreaders/hasmail/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkgconfig
+, gobject-introspection
+, pango
+, cairo
+, gtk2
+}:
+
+buildGoModule rec {
+  pname = "hasmail";
+  version = "2019-08-24";
+  name = "${pname}-${version}-unstable";
+
+  src = fetchFromGitHub {
+    owner = "jonhoo";
+    repo = "hasmail";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I've had reviews in the past which convinced me to not use `pname` in the `fetchFromGitHub` function. I don't have such a strong opinion about this but I'd like to be at least consistent in all of my _own_ PRs.

Reply:
Interesting, do you have any reference?

Reply:
https://github.com/NixOS/nixpkgs/pull/65480#discussion_r315799884